src/components/MultiHopTrade/hooks/useReceiveAddress.tsx

```diff
@@ -59,5 +64,6 @@ export const useReceiveAddress = () => {
     })()
   }, [buyAsset, dispatch, getReceiveAddressFromBuyAsset, setReceiveAddress])
 
-  return receiveAddress
+  // Always use the manual receive address if it is set
+  return manualReceiveAddress ?? receiveAddress
```

Comment:
Do we want to make this `||` for safety? i.e
<img width="117" alt="image" src="https://github.com/shapeshift/web/assets/17035424/a57c75a7-4c0d-4268-b4ec-0a728e9a86b8">

Reply:
Another good call, made that change in https://github.com/shapeshift/web/pull/4906/commits/985ddd9bee33e2d5c62a888344c64dc37f4ffbf8.